xc/device.ml

```diff
@@ -2462,10 +2463,12 @@ module Backend = struct
     end
 
     module DISK = struct
-      let max_emulated = Some 1
+      let max_emulated = Some 4
```

Comment:
We could potentially support an unlimited number of devices now (so .e.g 254?).
There have been some customers who would've wanted more than 4 emulated devices, but I didn't find a compelling reason to provide it (they'd be better served by installing PV drivers). Perhaps we could introduce an experimental way to override this, in another PR.